src/pandapipes/pf/result_extraction.py

```diff
@@ -212,6 +213,17 @@ def extract_branch_results_with_internals(net, branch_results, table_name,
 
             for i, (res_name, entry) in enumerate(res_mean_hydraulics):
                 res_table[res_name].values[pt] = res[i + 3][connected_ind] / num_internals
+        if len(res_branch) > 0:
+            use_numba = get_net_option(net, "use_numba")
+            indices, sections, connected_sum = _sum_by_group(use_numba, idx_pit, np.ones_like(idx_pit),
```

Comment:
Interesting enough, we do not need indices here anymore. You could replace it by _